README.md

```diff
@@ -0,0 +1,25 @@
+#Tetris: Survival of The Fittest
+
+A multiplayer tetris game.
+
+### Getting Started
+
+1. Clone the repository 
+  ```bash
+  git clone https://github.com/IncrementalMe/Tetris-Survival-Of-The-Fittest.git
+  ```
+
+2. Navigate to the project folder
+  ```bash
+  cd Tetris-Survival-Of-The-Fittest/tetris
+  ```
+
+3. Install dependencies
+  ```bash
+  npm install
+  ```
+
+4. Start the game server
+  ```bash
+  node index.js
+  ```
```

Comment:
Good to be explicit about browser support, for small project latest version of chrome only is surprisingly standard.